### PR TITLE
CSS Library: add feedback-related foreground tokens

### PR DESCRIPTION
**Pull request by narin**
## Chromatic

https://va-mobile-feedback-colors--65a6e2ed2314f7b8f98609d8.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/va-mobile-library/issues/541

The VA Mobile DS team would like to add new feedback-related foreground tokens to the system and need 2 new primitive and 2 new semantic tokens added to css-library. This PR:

- Updates `copy-uswds-color-tokens` script to include the following primitive tokens:
  - uswds-system-color-green-cool-vivid-30
  - uswds-system-color-gold-vivid-40
- Adds the following semantic that references the newly added uswds colors
  - vads-color-feedback-foregound-success-on-dark: uswds-system-color-green-cool-vivid-30
  - vads-color-feedback-foregound-warning-on-light: uswds-system-color-gold-vivid-40

## QA Checklist
N/A

## Screenshots
N/A

## Acceptance criteria
N/A

## Definition of done
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by jamigibbs — COMMENTED** [review]
Hi @narin Can you also run `yarn build` and checkin the generated files?

I've also added the DST designers to review naming and color mapping. We will probably want to coordinate updating documentation as well.

File: packages/css-library/tokens/color.json [hunk 1]
```diff
@@ -267,6 +267,12 @@
   "vads-color-action-border-base-active-on-dark": {
     "*": { "value": "{uswds-system-color-gray-30}"}
   },
+  "vads-color-feedback-foregound-success-on-dark": {
```
Comment (jamigibbs) [inline packages/css-library/tokens/color.json:270]: I think `foregound` should be `foreground`?

Comment (narin) [reply packages/css-library/tokens/color.json]: Nice catch! Fixed.

**Review by danbrady — APPROVED** [review]
Ok, added tokens to Airtable. I'll create another ticket to update the doc site.

**Comment by jessicawoodin** [comment]
> Hi @narin - I noticed you're adding two new tokens, but I don't see the same tokens for the alternate modes? You're requesting:
> 
> * vads-color-feedback-foregound-success-**on-dark**
> * vads-color-feedback-foregound-warning-**on-light**
> 
> Should we also have:
> 
> * vads-color-feedback-foregound-success-**on-light**
> * vads-color-feedback-foregound-warning-**on-dark**

Hi @danbrady - We can add the corresponding tokens for light/dark mode if you think that's best! In the past, I think we've only added the semantic tokens for any missing primitive tokens that were needed for the mobile app. If we want to add them, they would be:

* vads-color-feedback-foregound-success-on-light :arrow_right: uswds-system-color-green-cool-vivid-50
* vads-color-feedback-foregound-warning-on-dark :arrow_right: uswds-system-color-gold-vivid-30

**Comment by narin** [comment]
Thanks @jessicawoodin @danbrady. Added tokens for the alternate modes and rebuilt.

**Comment by danbrady** [comment]
Thanks, @jessicawoodin @narin - I've updated the Airtable and the ticket to add them to the doc site.

**Review by danbrady — APPROVED** [review]

**Pull request merged**